VortexEngine/src/Menus/Menu.cpp

```diff
@@ -23,6 +24,56 @@ Menu::~Menu()
 {
 }
 
+LedMap ledPermutations[] = {
```

Comment:
put this definition at the top of the file (above the Menu::Menu definition but below the header includes line 10-11)

Also make this static, just put the static keyword in front of the type, that just means it's a global that should only be used in this file and not from outside the file